src/Entity/IdeasWorkshop/Idea.php

```diff
@@ -202,6 +222,7 @@ class Idea implements AuthorInterface, ReportableInterface, EnabledInterface
     use EntityTimestampableTrait;
     use EntityNameSlugTrait;
 
+    public const EXTEND_COUNT_LIMIT = 2;
```

Comment:
public to add detail in the test here https://github.com/EnMarche/en-marche.fr/pull/3446/files#diff-468229cd6b82697c76fb1bdc3a1159a4R21